bool start
string bagFilePath
float32 timeScaling
---
bool success
string message